docs/DEFAULT-VPC.md

```diff
@@ -15,58 +30,205 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-# Default VPC
+## General
+
+The creation of a default VPC network is enabled by default and can be disabled
+via `skip_default_vpc_creation` input parameter. One of the main differences
+to the default VPC created by Google is the network range used. This module uses
+RFC 1918\] ip ranges from `172.16.0.0/12` for primary ranges while Google uses
+ranges from `10.0.0.0/8` as this prefix is used widely used within Metros
+internal network. This should help users to easily separate VPCs with on-premise
+connectivity from those without.
+
+## Supported regions
+
+The module aims to support all main regions within the European Union:
+
+| Region            | Location                   |
+| ----------------- | -------------------------- |
+| europe-west1      | St. Ghislain, Belgium, EU  |
+| europe-west3      | Frankfurt, Germany EU      |
+| europe-west4      | Eemshaven, Netherlands, EU |
+| europe-west8      | Milan, Italy, EU           |
+| europe-west9      | Paris, France, EU          |
+| europe-north1     | Hamina, Finland, EU        |
+| europe-central2   | Warsaw, Poland, EU         |
+| europe-southwest1 | Madrid, Spain, EU          |
+
+## IP configurations
+
+### RFC1918 free to use IP address ranges in the default VPC
+
+#### 10.0.0.0 - 10.255.255.255  (10/8 prefix)
+
+We will not use `10.192.0.0/10` for any further implementation. If you plan
+to add additional network features to the default VPC use any IP address
+range from here.
```

Comment:
Isn't this a bit confusing when considered together with what is stated in line 37 and following?

Reply:
Changed that part a bit, is it more clear this way?

Reply:
yup